packages/notifications/src/elements/content/Title.tsx

```diff
@@ -8,14 +8,19 @@
 import React, { HTMLAttributes } from 'react';
 import { StyledTitle } from '../../styled';
 
+export interface ITitleProps extends HTMLAttributes<HTMLDivElement> {
+  /** Style using regular (non-bold) font weight */
+  isRegular?: boolean;
```

Comment:
Even if it's optional, should we include a `defaultProp` of `false` show it's shown correctly in the generated prop-sheets?

Reply:
@austin94 that isn't consistent with any of our other boolean props. Consider https://garden.zendesk.com/react-components/forms/#label, for example.

Reply:
👍 Sounds good to me